polymesh_schema.json

```diff
@@ -19,6 +19,13 @@
             "doc_type": "Option<DocumentType>",
             "filing_date": "Option<Moment>"
         },
+        "Version": {
```

Comment:
Might need different version enums here? Is the plan to just have one and include the largest number of all version enums? Won't work if we use more semantic version variant names.

Reply:
There is not external access to update `Version`, but our CLI tests were failing due to its missing definition in the schema file.

Reply:
Hmm; in that case, can we get away with an empty `"Version": {}`?

Reply:
New version with `u8`

Reply:
New version with `u8`.